modules/godot/register_types.cpp

```diff
@@ -132,30 +137,40 @@ void initialize_godot_module(ModuleInitializationLevel p_level) {
 
 		ClassDB::register_class<SharedComponentResource>();
 
+#ifdef DEBUG_ENABLED
+		// TS is nullptr at this point when running tests in debug mode, and it causes a crash. I did not find a better way to check for that.
+		if (TS != nullptr) {
+			ECS::preload_scripts();
+		}
+#else
+		ECS::preload_scripts();
+#endif
 		memnew(ScriptEcs());
 
 	} else if (p_level == MODULE_INITIALIZATION_LEVEL_EDITOR) {
 		if (Engine::get_singleton()->is_editor_hint()) {
 			EditorNode::add_init_callback(_editor_init);
 
-			if (Node3DEditor::get_singleton() != nullptr) {
-				// Add component gizmos:
-				Node3DEditor::get_singleton()->add_gizmo_plugin(Ref<Components3DGizmoPlugin>(Components3DGizmoPlugin::get_singleton()));
-			}
-
 			// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Gizmos
 			Components3DGizmoPlugin::get_singleton()->add_component_gizmo(memnew(TransformComponentGizmo));
 			Components3DGizmoPlugin::get_singleton()->add_component_gizmo(memnew(MeshComponentGizmo));
 
 		} else {
 			// Load the Scripted Components/Databags/Systems
-			ScriptEcs::get_singleton()->register_runtime_scripts();
+			// ScriptEcs::get_singleton()->register_runtime_scripts();
```

Comment:
is this meant to be disabled?

Reply:
As far as I remember it caused problems at that point, and we agreed to comment it out for now.

Reply:
I can check the extra whitespace ones, as long as it passes the static checks